### DISCUSSION TITLE
Are there any realms open to new characters other than PTR?

### DISCUSSION THREAD

**Discussion by AppalachianMounta1n** [chromiecraft/chromiecraft#7565] [General]
I only see PTR and Chromiecraft RP, but can only make a PTR character

**Comment by Stump876** [comment]
Chromiecraft RP is the live server. I'm going to make a guess and say you only tried rolling a Death Knight. They are currently locked on CC live, but open for test in the PTR.

**Reply by AppalachianMounta1n** [reply]
Yeah that would do it haha, thank you!
